pkg/kubelet/kubelet_pods.go

```diff
@@ -695,7 +695,7 @@ func (kl *Kubelet) makeEnvironmentVariables(pod *v1.Pod, container *v1.Container
 	for _, envVar := range container.Env {
 		runtimeVal := envVar.Value
 		if runtimeVal != "" {
-			// Step 1a: expand variable references
+			// Step 1a: expand variable references, if extended variables is `runtimeVal`, the extension is ordered
```

Comment:
I do not follow this comment. What extension? Did you mean expansion? Also grammar unclear, It says "variables" (plural), but refer to this singular value. Did you mean other variables? or this one?

Please clarify.